.python-version

```diff
@@ -0,0 +1 @@
+3.10.11
```

Comment:
I guess this is why CI failing. we should ignore those files from IDE. we could add it to the .gitignore.

Reply:
Oops, my bad.